src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoDayNames.cs

```diff
@@ -10,92 +10,30 @@ namespace System.Globalization.Tests
         }
 
-        // PosTest2: Call DayNames setter method should return correct value
         [Fact]
-        public void TestSetter()
+        public void DayNames_Set()
         {
-            VerificationHelper(new DateTimeFormatInfo(),
-                    new string[] {
-                    "1",
-                    "2",
-                    "3",
-                    "4",
-                    "5",
-                    "6",
-                    "7"
-                    },
-                    true);
+            string[] newDayNames = new string[] { "1", "2", "3", "4", "5", "6", "7" };
+            var format = new DateTimeFormatInfo();
+            format.DayNames = newDayNames;
+            Assert.Equal(newDayNames, format.DayNames);
         }
 
-        // NegTest1: ArgumentNullException should be thrown when The property is being set to a null reference
         [Fact]
-        public void TestNull()
+        public void DayNames_Set_Invalid()
         {
-            Assert.Throws<ArgumentNullException>(() =>
-            {
-                new DateTimeFormatInfo().DayNames = null;
-            });
-            Assert.Throws<ArgumentNullException>(() =>
-            {
-                new DateTimeFormatInfo().DayNames = new string[] {
-                    "1",
-                    "2",
-                    "3",
-                    null,
-                    "5",
-                    "6",
-                    "7"
-                };
-            });
-        }
-
-        // NegTest2: ArgumentException should be thrown when The property is being set to an array that is multidimensional or whose length is not exactly 7
-        [Fact]
-        public void TestInvalidArray()
-        {
-            Assert.Throws<ArgumentException>(() =>
-            {
-                new DateTimeFormatInfo().DayNames = new string[] { "sun" };
-            });
-        }
-
-        // NegTest3: InvalidOperationException should be thrown when The property is being set and the DateTimeFormatInfo is read-only
-        [Fact]
-        public void TestReadOnly()
-        {
-            Assert.Throws<InvalidOperationException>(() =>
-            {
-                DateTimeFormatInfo.InvariantInfo.DayNames = new string[] { "1", "2", "3", "4", "5", "6", "7" };
-            });
-        }
-
-        private void VerificationHelper(DateTimeFormatInfo info, string[] expected, bool setter)
-        {
-            if (setter)
-            {
-                info.DayNames = expected;
-            }
+            Assert.Throws<ArgumentNullException>(() => new DateTimeFormatInfo().DayNames = null); // Value is null
+            Assert.Throws<ArgumentNullException>(() => new DateTimeFormatInfo().DayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
+            Assert.Throws<ArgumentException>(() => new DateTimeFormatInfo().DayNames = new string[] { "sun" }); // Value.Length is not 13
```

Comment:
Another 13 v 7 comment discrepancy.  Maybe we should just reshape the week to fit... so long as we keep a 5 day work week... nah, we'd have to name the new days, and that'll be stuck in committee forever.

Anyways, not going to point out any more of this discrepancy, but with this one so far from the first it seemed like a good "there might be more of these" moment.